Gemfile

```diff
@@ -5,9 +5,9 @@ platforms :ruby do
 end
 
 group :test do
-  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 4.2'])
+  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 5.0'])
```

Comment:
Lifted from @masongup-mdsol's PR. Why would we not want to ensure it works with different AR versions? We use primarily Rails 3 and Rails 4 at this company.